src/playground-code-editor.ts

```diff
@@ -505,21 +506,31 @@ export class PlaygroundCodeEditor extends LitElement {
       }
     });
 
-    cm.on('renderLine', (_editorInstance: Editor, line, elt) => {
-      if (!this.lineWrapping) {
-        return;
-      }
+    this._indentingLine(cm);
 
-      const basePadding = 4;
-      const charWidth = cm.defaultCharWidth();
-      const tabSize = cm.getOption('tabSize') || basePadding;
-      const off = CodeMirror.countColumn(line.text, null, tabSize) * charWidth;
+    this._codemirror = cm;
+  }
 
-      elt.style.textIndent = `-${off}px`;
-      elt.style.paddingLeft = `${basePadding + off}px`;
-    });
+  private _indentingLine(cm: CodeMirror.Editor) {
+    if (this.lineWrapping) {
```

Comment:
I think I would actually just inline this directly in the `lineWrapping` case in `update`.

Reply:
The `lineWrapping` case only happens when the property changes. If it is initially set as true, then `renderLine` never runs unless it is also in the `_createView`.

Reply:
Sorry I mean this entire function could be removed, and both cases put inside the switch block. Just because the function is so short that it doesn't really seem to help readability.